tests/spread/extensions/gnome-3-38/task.yaml

```diff
@@ -0,0 +1,58 @@
+summary: Build and run a basic gnome snap using extensions
+
+# The content snap required for the test to succeed is only
+# available on a subset of all the architectures this testbed
+# can run on.
+systems:
+  - ubuntu-20.04
+  - ubuntu-20.04-64
+  - ubuntu-20.04-amd64
+  - ubuntu-20.04-arm64
+  - ubuntu-20.04-i386
+  - ubuntu-20.04-armhf
+
+environment:
+  SNAP_DIR: ../snaps/gtk3-hello
+
+prepare: |
+  #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  set_base "$SNAP_DIR/snap/snapcraft.yaml"
+
+restore: |
+  cd "$SNAP_DIR"
+  snapcraft clean
+  rm -f ./*.snap
+
+  #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  restore_yaml "snap/snapcraft.yaml"
+
+execute: |
+  cd "$SNAP_DIR"
+  sed -i 's/gnome-3-28/gnome-3-38/' "snap/snapcraft.yaml"
+  sed -i 's/core18/core20/' "snap/snapcraft.yaml"
```

Comment:
Two options:

- sed command to be usr/local/bin/gtk3-hello
- sed so that prefix is set in the part (or globally without sedding, it should not affect previous bases)